test/fixtures/es-modules/package-cjs-named-error/default-and-renamed-import.mjs

```diff
@@ -0,0 +1,4 @@
+import abc, {
+  comeOn as comeOnRenamed,
+  everybody,
+} from './fail.cjs';
```

Comment:
Additional test cases that come to my mind:

* A test case where a valid import comes before the failing one:

```js
import fs from 'fs';
import { comeOn } from './fail.cjs';
```

* A very long import statement, that should be rendered on multiple lines:

```js
import {
  comeOn,
  everybody,
  comeOn1,
  comeOn2,
  comeOn3,
  comeOn4,
  comeOn567,
} from './fail.cjs';
```

which would render as

```js

Reply:
Maybe have one with a comment in the declaration:

```js
import { we, will,
  //     we, will,
         rock, you } from './queen.cjs';
```

Reply:
Are you not interested in adding this test ⬆️?